src/main/java/mate/academy/model/Movie.java

```diff
@@ -0,0 +1,56 @@
+package mate.academy.model;
+
+import javax.persistence.Entity;
+import javax.persistence.GeneratedValue;
+import javax.persistence.GenerationType;
+import javax.persistence.Id;
+
+@Entity
+public class Movie {
+    @Id
+    @GeneratedValue(strategy = GenerationType.IDENTITY)
+    private Long id;
+    private String title;
+    private String description;
+
+    public Movie() {
+    }
+
+    public Movie(String title, String description) {
```

Comment:
Do you really need this constructor?
you don't use it

Reply:
Ok, I will use it in the main method